pom.xml

```diff
@@ -212,7 +212,11 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
```

Comment:
`mvn install` is fine as we have CI enabled. Most recent check on PR is successful too - https://github.com/Hygieia/hygieia-core/actions/runs/279503762. You can directly download the jar file from maven central if you don't need to make changes to [hygieia-core](https://github.com/hygieia/hygieia-core)